ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2

```diff
@@ -62,6 +62,32 @@ router isis {{ router_isis.instance }}
 {%     endif %}
 {%     if router_isis.advertise.passive_only is arista.avd.defined(true) %}
    advertise passive-only
+{%     endif %}
+{%     if router_isis.spf_interval.interval is arista.avd.defined %}
+{%         if router_isis.spf_interval.wait_interval is arista.avd.defined %}
+   spf-interval {{ router_isis.spf_interval.interval }} {{ router_isis.spf_interval.wait_interval }}
+{%         else %}
+   spf-interval {{ router_isis.spf_interval.interval }}
+{%         endif %}
+{%     endif %}
+{%     if router_isis.graceful_restart is arista.avd.defined %}
+{%         if router_isis.graceful_restart.enabled is arista.avd.defined(true) %}
+   graceful-restart
+{%         endif %}
+{%         if router_isis.graceful_restart.restart_hold_time is arista.avd.defined %}
+   graceful-restart restart-hold-time {{ router_isis.graceful_restart.restart_hold_time }}
+{%         endif %}
+{%         if router_isis.graceful_restart.t2.wait_time is arista.avd.defined %}
+   graceful-restart t2 level-1 {{ router_isis.graceful_restart.t2.wait_time }}
+   graceful-restart t2 level-2 {{ router_isis.graceful_restart.t2.wait_time }}
+{%         else %}
```

Comment:
Please remove the wait_time option. Even though the CLI includes this abstraction, we should not replicate it here. Keep the level-1 and level-2

Reply:
Fixed!